### PR TITLE
Fix: update scope criteria for analytics packages

### PR DESCRIPTION
**Pull request by lwasser**
This pr updates our scope statement found in the peer review guide to specifically address analytics related packages. It is now open for community review! 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206952129256524

### CODE REVIEW

File: about/package-scope.md [hunk 1]
```diff
@@ -35,43 +40,32 @@ data processing and the various stages of managing the
...
 pyOpenSci. In addition to fitting into one or more of these categories, your package should have some level of
 demonstrated scientific application. This could be a use case that you can
 link to or a tutorial that demonstrates its potential application for science.
 
-Below we provide examples of packages from pyOpenSci ecosystem. Since we
-have a growing community of packages, in some cases we will link to R packages
-within the rOpenSci community that match the category scope for reference.
+Below we provide examples of packages from pyOpenSci ecosystem.
 
 We will update this page as our review process evolves.
```
Comment (lwasser) [inline about/package-scope.md:71]:
```suggestion
```

redundant! this was in here 3 times. 😆

File: about/package-scope.md [hunk 2]
```diff
@@ -167,6 +159,34 @@ analyzing data.
...
+We consider the following when determining whether an analytics-related package is within our review scope:
+
+1. If your package facilitates a scientist using a **known or vetted statistical, AI or  Analytical approach** we consider that in-scope. Before submitting to us, please ensure that your package's documentation directs users to existing paper(s) or pre-print(s) that document that approach's application. Further, be sure to link to these publications in your package review submission.
+
+The review for this package:
+
+* requires at least 1 domain specialist
+* will never vet the analytical method itself.
+
+2. If your package introduces a novel or newer analytic approach that is not yet known/ vetted/ accepted, we can not review it. We cannot review projects that exist as a proof-of-concept demonstration of a model or analytical approach that might accompany a paper. In this case, the approach should be sent to a scientific journal for vetting.
```
Comment (isabelizimm) [inline about/package-scope.md:183]:
```suggestion
2. If your package introduces a novel or newer analytic approach that is not yet vetted/ accepted by a scientific journal, we can not review it. We cannot review projects that exist as a proof-of-concept demonstration of a model or analytical approach that might accompany a paper. In this case, the approach should be sent to a scientific journal for vetting.
```
nit: "known" might be different for different people

**Review by isabelizimm — COMMENTED** [review]
a few nits, but overall I think this hits the notes we need it to in order to clarify scope around analytic packages 🙌

File: about/package-scope.md [hunk 3]
```diff
@@ -167,6 +159,34 @@ analyzing data.
  * Example: [Example from rOpenSci - rrlite](https://github.com/ropensci/software-review/issues/6)
 
 
+## Scope for packages that support analytics, statistics and modeling
+
+The mission of pyOpenSci's open peer review process is to:
```
[the remark this answers is not in the document]
Comment (lwasser) [reply about/package-scope.md:164]: good point! yes i moved it!!

File: about/package-scope.md (hunk 2 above)
Comment (lwasser) [reply about/package-scope.md:183]: great catch!

**Comment by lwasser** [comment]
@isabelizimm thank you so much for the review! 🙌  i've incorporated your suggestions and am going to merge this now!!

**Pull request merged**